### PR TITLE
Envoy integration documentation

### PR DESCRIPTION
**Pull request by sjyothi54**
**Ticket:** https://new-relic.atlassian.net/browse/NR-231688
**Description:** Created Envoy integration documentation
**Screenshot:**
screencapture-localhost-8000-docs-infrastructure-host-integrations-host-integrations-list-envoy-integration-2024-03-07-15_03_54

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by bradleycamacho** [comment]
Hi @sjyothi54 ! Since this is a brand new doc, I'll have to bring this into an upcoming sprint to review and publish. I can get it into our March 18th sprint if that works!

**Comment by sjyothi54** [comment]
> Hi @sjyothi54 ! Since this is a brand new doc, I'll have to bring this into an upcoming sprint to review and publish. I can get it into our March 18th sprint if that works!

@bradleycamacho We're okay with reviewing this document in the next sprint, but please ensure there are no delays, as this quickstart needs to go live by the end of the month.

**Comment by bradleycamacho** [comment]
Netlify build

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/envoy-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+To use the Envoy integration, you need to also [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. The infrastructure agent monitors the host itself, while the integration you'll install in the next step extends your monitoring with Envoy-specific data.
+    </Step>
+    <Step>
+
+## Enable the Envoy integration
+
+To set up the Envoy integration, follow these steps:
+
+1. Create a file named `nri-prometheus-config.yml` in the integrations directory:
```
Comment (bradleycamacho) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/envoy-integration.mdx:35]: Would this be the correct phrasing?

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/envoy-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+title: Envoy integration
+tags:
+    - New Relic integrations
+    - Envoy integration
+metaDescription: Use New Relic infrastructure agent to get a dashboard with metrics from your Envoy.
+freshnessValidatedDate: 03/21/2024
+---
+import envoyDashboard from 'images/infrastructure_screenshot_full-envoy-dashboard.webp'
+
+Gain deep insights into Envoy performance with seamless data integration into New Relic. Monitor key metrics to ensure the optimal performance of your Envoy clusters. Create alerts to stay ahead of spikes, build custom dashboards for tailored views, and proactively optimize your database monitoring.
```
Comment (bradleycamacho) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/envoy-integration.mdx:11]: @sjyothi54 could you help me rewrite this intro? I lack the knowledge about envoy to talk about specifics

**Comment by bradleycamacho** [comment]
Hi @sjyothi54! I made some changes to the doc and left a few comments. I'd love if you could review it and make any changes or leave any comments for me.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/envoy-integration.mdx (hunk 1 above)
Comment (sjyothi54) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/envoy-integration.mdx]: @bradleycamacho I have rephrased.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/envoy-integration.mdx (hunk 2 above)
Comment (sjyothi54) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/envoy-integration.mdx]: @bradleycamacho Rephrased the description.

**Comment by sjyothi54** [comment]
> Hi @sjyothi54! I made some changes to the doc and left a few comments. I'd love if you could review it and make any changes or leave any comments for me.

Hi @bradleycamacho Thank you for reviewing PR, I updated the changes. Kindly review it

**Pull request merged**